producer/ue_authentication.go

```diff
@@ -86,7 +86,8 @@ func HandleUeAuthPostRequest(request *http_wrapper.Request) *http_wrapper.Respon
 }
 
 // func UeAuthPostRequestProcedure(updateAuthenticationInfo models.AuthenticationInfo) (
-//    response *models.UeAuthenticationCtx, locationURI string, problemDetails *models.ProblemDetails) {
+//
+//	response *models.UeAuthenticationCtx, locationURI string, problemDetails *models.ProblemDetails) {
```

Comment:
@gruyaume, actually, should we remove all these lines?

Reply:
Please read my reply above.